diagnostic reports/README.md

```diff
@@ -0,0 +1,374 @@
+| component / CompoundStatement                             | mapped from the [Test Group Header](../observations/README.md#Test-Group-Header-(JSON-FHIR->-XML-HL7)) or [Test Result](../observations/README.md#Test-Result-(JSON-FHIR->-XML-HL7)) Observations referenced in `DiagnosticReport.result` where `Observation.specimen` references the `Specimen` |
+
+<details>
+ <summary>Example XML</summary>
+
+```XML
+
+<CompoundStatement classCode="CLUSTER" moodCode="EVN">
+ <id root="D6930FE2-0417-448B-85FA-0ACA7284CBD0"/>
+ <code code="123038009" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="specimen (specimen)"/>
+ <statusCode code="COMPLETE"/>
+ <effectiveTime>
+  <center nullFlavor="NI"/>
+ </effectiveTime>
+ <availabilityTime value="20030628112300"/>
+ <specimen typeCode="SPC">
+  <specimenRole classCode="SPEC">
+   <id root="DB2E5F7D-B0D0-488A-825A-BE58D09953CF"/>
+   <id extension="C3179539H" root="2.16.840.1.113883.2.1.4.5.2"/>
+   <effectiveTime>
+    <center value="20030627000000"/>
+   </effectiveTime>
+   <specimenSpecimenMaterial classCode="MAT" determinerCode="INSTANCE">
+    <desc>Plasma</desc>
+   </specimenSpecimenMaterial>
+  </specimenRole>
+ </specimen>
+ <component contextConductionInd="true" typeCode="COMP">
+  <NarrativeStatement classCode="OBS" moodCode="EVN">
+   <id root="DF0FEFF0-32C1-4BB9-8DDD-76E2D13B5D0F"/>
+   <text mediaType="text/x-h7uk-pmip">CommentType:LAB SPECIMEN COMMENT(E271)
+CommentDate:20030627000000
+
+Quantity: 1750.000 mL
+Received Date: 2003-06-27 18:24
+   </text>
+   <statusCode code="COMPLETE"/>
+   <availabilityTime value="20030628112300"/>
+  </NarrativeStatement>
+ </component>
+ <component contextConductionInd="true" typeCode="COMP">
+  <CompoundStatement classCode="BATTERY" moodCode="EVN">
+   
+   ...
+   
+  </CompoundStatement>
+ </component>
+</CompoundStatement>
+```
+</details>
+
+5. Where `extension[index].url` equals `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid"`, 
+`extension[index2].url` equals `"descriptionDisplay"` and `extension[index2].value` does not equal `Specimen.type.coding[0].display` 
+
+### Specimen Level Narrative Statements
+
+A Narrative Statement is created to preserve data from FHIR fields that have no direct mapping.
+
+| Mapped to (XML HL7 NarrativeStatement element) | Mapped from (JSON FHIR / other source ) |
+|------------------------------------------------|-----------------------------------------|
+| id [@root]                                     | Unique ID generated by the adaptor      |
+| text [@mediaType]                              | fixed value = `"text/x-h7uk-pmip"`      |
+| text                                           | EDIFACT comment, as described below     |
+| statusCode                                     | fixedValue = `"COMPLETE"`               |
+| availabilityTime [@value]                      | `DiagnosticReport.issued`               |
+
+#### Text Element Mapping
```

Comment:
Minor: Do we want to be clear that this Narrative Statement is optionally generated, depending on whether any of additional FHIR fields are populated.

Reply:
Yes, I've rephrased it.